config/session.php

```diff
@@ -29,9 +29,9 @@
     |
     */
 
-    'lifetime' => 120,
+    'lifetime' => 1,
```

Comment:
1 what? Minute?

Reply:
Ah. no. that would b bad. lets set it back to 60.